sample/src/main/java/com/willowtreeapps/signinwithapple/sample/SampleJavaActivity.java

```diff
@@ -35,8 +35,8 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         SignInWithAppleCallback callback = new SignInWithAppleCallback() {
             @Override
-            public void onSignInWithAppleSuccess(@NonNull String authorizationCode) {
-                Toast.makeText(SampleJavaActivity.this, authorizationCode, LENGTH_SHORT).show();
+            public void onSignInWithAppleSuccess(@NonNull String authorizationCode, @NonNull String idToken) {
+                Toast.makeText(SampleJavaActivity.this, authorizationCode + " " + idToken, LENGTH_SHORT).show();
```

Comment:
Why we show auth code and token in toast?

Reply:
This is just a sample activity for the library